test/hexpm_web/controllers/api/organization_controller_test.exs

```diff
@@ -103,4 +103,43 @@ defmodule HexpmWeb.API.OrganizationControllerTest do
       assert result["errors"] == "number of seats cannot be less than number of members"
     end
   end
+
+  describe "GET /api/orgs/:organization/audit_logs" do
+    test "returns the first page of audit_logs related to this organization when params page is not specified",
+         %{user1: user1, organization: organization} do
+      insert(:organization_user, organization: organization, user: user1, role: "read")
+
+      insert(:audit_log, action: "organization.test", organization: organization)
+
+      conn =
+        build_conn()
+        |> put_req_header("authorization", key_for(user1))
+        |> get("api/orgs/#{organization.name}/audit_logs")
+
+      assert [
+               %{
+                 "action" => "organization.test"
+               }
+             ] = json_response(conn, :ok)
```

Comment:
Can we single-line this?

Reply:
Fixed in 495b8405 Single-line assertions in audit_logs related controller tests

I also single-lined other assertions like this, please check